components/Label.js

```diff
@@ -0,0 +1,13 @@
+import { styled } from "styled-components";
+
+const Label = styled.span`
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+  background-color: white;
+  padding: 0.25rem 0.5rem;
+  border-radius: 1rem;
+  text-transform: uppercase;
+`;
+
+export default Label;
```

Comment:
Another possibility for a default export of a styled component is:

```jsx
export default styled.span`
  position: absolute;
  top: 1rem;
  right: 1rem;
  background-color: white;
  padding: 0.25rem 0.5rem;
  border-radius: 1rem;
  text-transform: uppercase;
`;
```

Should work the same if you do not want to use named imports. Just a information for you. :)